packages/react-native-client/README.md

```diff
@@ -0,0 +1,402 @@
+   class MembraneBroadcastSampleHandler: RPBroadcastSampleHandler {
+       let broadcastSource = BroadcastSampleSource(appGroup: appGroup)
+       var started: Bool = false
+
+
+       override func broadcastStarted(withSetupInfo _: [String: NSObject]?) {
+           started = broadcastSource.connect()
+
+           guard started else {
+               os_log("failed to connect with ipc server", log: logger, type: .debug)
+
+               super.finishBroadcastWithError(NSError(domain: "", code: 0, userInfo: nil))
+
+               return
+           }
+
+           broadcastSource.started()
+       }
+
+       override func broadcastPaused() {
+           broadcastSource.paused()
+       }
+
+       override func broadcastResumed() {
+           broadcastSource.resumed()
+       }
+
+       override func broadcastFinished() {
+           broadcastSource.finished()
+       }
+
+       override func processSampleBuffer(_ sampleBuffer: CMSampleBuffer, with sampleBufferType: RPSampleBufferType) {
+           guard started else {
+               return
+           }
+
+           broadcastSource.processFrame(sampleBuffer: sampleBuffer, ofType: sampleBufferType)
+       }
+   }
+   ```
+
+   Replace `{{GROUP_IDENTIFIER}}` and `{{BUNDLE_IDENTIFIER}}` with your group
+   identifier and bundle identifier respectively.
+
+6. In project's Podfile add the following code:
+   ```rb
+   target 'MembraneScreenBroadcastExtension' do
+     pod 'MembraneRTC/Broadcast'
+   end
+   ```
+   > This new dependency should be added outside of your application target.
+   > Example
+   >
+   > ```rb
+   > target 'ReactNativeMembraneExample' do
+   >  ...
+   > end
+   >
+   > target 'MembraneScreenBroadcastExtension' do
+   >  pod 'MembraneRTC/Broadcast'
+   > end
+   > ```
+7. Run `pod install` in your `ios/` directory
+8. Add the following constants in your Info.plist:
+   ```xml
+   <key>AppGroupName</key>
+   <string>{{GROUP_IDENTIFIER}}</string>
+   <key>ScreencastExtensionBundleId</key>
+   <string>{{BUNDLE_IDENTIFIER}}.MembraneBroadcastSampleHandler</string>
+   ```
+   Replace `{{GROUP_IDENTIFIER}}` and `{{BUNDLE_IDENTIFIER}}` with your group
+   identifier and bundle identifier respectively.
+9. Rebuild the app and enjoy!
+
+# Example
+
+We strongly recommend checking out our example app that implements a basic video
+room client. To run the app:
+
+1. Go to Membrane's server demo repo:
+   https://github.com/fishjam-dev/fishjam-videoroom. Follow instructions there
+   to setup and run demo server.
+2. Clone the repo
+3. ```
+   cd example
+   yarn
+   ```
+4. In App.ts replace server url with your server's url.
+5. `yarn run android` or `yarn run ios` or run project from Android Studio /
+   Xcode just like every RN project. Note that simulators won't work, you have
+   to test on real device for camera and screensharing to run.
+
+# Usage
+
+> [!CAUTION] This part of documentation is out of date. It will be updated soon.
```

Comment:
```suggestion
> [!CAUTION] 
> This part of documentation is out of date. It will be updated soon.
```
Note that GH has special formatting for this syntax, when it is splitted into two lines